.eslintrc.js

```diff
@@ -0,0 +1,180 @@
+// Standard Formatting rules for Kolbot scripts
+// Compatible with ESLint plugin
+
+module.exports = {
+	"root": true,
+	"extends": "eslint:recommended",
+	"parserOptions": {
+		"ecmaFeatures": {
+			"impliedStrict": true,
+		}
+	},
+	"env": {
+		"es6": true,
+	},
+	"globals": {
```

Comment:
It would be nice to include a short explanation about all of the globals and where they come from.

`// The following globals are defined from the D2Bot list.`